### DISCUSSION TITLE
Is it heroicons.dev or heroicons.com

### DISCUSSION THREAD

**Discussion by SagnikPradhan** [tailwindlabs/heroicons#923] [General]
Sorry, I keep getting confused

**Comment by daylmo** [comment]
I hope heroicons.dev belongs to a third party

**Comment by zefir-git** [comment]
Does it matter? heroicons.dev also has the same icons (or at least very similarly looking ones designed by the same person and also MIT-licensed). And it does seem that heroicons.dev also gives you way more options. Otherwise afaik herocions.com is the official website. However if both work for you I don't see why you can't use them.
